src/bt_peer_connection.cpp

```diff
@@ -2306,23 +2298,14 @@ namespace libtorrent {
 		TORRENT_ASSERT(m_sent_bitfield);
```

Comment:
The two asserts above can be removed I believe, right? (since they're done in send_simple_message())